From b0561a0317912d38653c0b8407890ddcb86009e3 Mon Sep 17 00:00:00 2001
From: Eric Cooper <ecc@cmu.edu>
Date: Thu, 12 Dec 2013 14:06:02 -0500
Subject: [PATCH 1/2] change method of passing options to curl to avoid FHS
 violation (closes: #732021)

Signed-off-by: Eric Cooper <ecc@cmu.edu>
---
 doc/FAQ | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/doc/FAQ b/doc/FAQ
index bcd0c51..142cbe8 100644
--- a/doc/FAQ
+++ b/doc/FAQ
@@ -81,8 +81,8 @@ Passing environment variables to approx
 
 Passing options to the curl(1) command used by approx
 
-  You can specify options in a .curlrc file in the approx user's home directory
-  (/var/cache/approx/ by default).
+  Use the above method to pass the CURL_HOME environment variable to approx,
+  and use the file $CURL_HOME/.curlrc to specify the desired curl options.
 
   For example, adding the line "--ipv4" to this file will force curl to
   resolve hostnames to IPv4 addresses only.
-- 
1.8.5.1

